### ISSUE TITLE
HTML indention does not work correctly in eex and leex files

### ISSUE THREAD

**Issue by bdtomlin** [elixir-lsp/vscode-elixir-ls#162]
### Environment

Most of this can be filled out by running the VSCode command (by default bound to Ctrl+Shift+P) "ElixirLS: Copy Debug Info"

- Elixir & Erlang versions (elixir --version): Erlang/OTP 23, Elixir 1.11.3   
- VSCode ElixirLS version: v0.6.2
- Operating System Version: macOS Big Sur 11.1

### Troubleshooting

- [x] Restart your editor (which will restart ElixirLS) sometimes fixes issues
- [x] Stop your editor, remove the entire `.elixir_ls` directory, then restart your editor

HTML indenting doesn't work correctly in html.eex and html.leex files.

In an html.eex file using emmet if you type div<tab> you will end up with `<div>|</div>` where | is the cursor. If you hit enter you end up with the following where `|` is the cursor:

```html
<div>
|</div>
```

If you change the file format to html and perform the same actions you get the following, which is also expected in the first example:
```html
<div>
  |
</div>
```

Basically, the file is not utilizing html indention. FWIW, erb files have the same problem. PHP files work correctly so it is possible. I don't know anything about VSCode extensions, but I will try to dig in and figure out what the issue is at some point if no one else knows what is going on.

**Comment by axelson** [comment]
This is a subset of #98 which has some links to additional information if you'd like to dive in. Unfortunately https://github.com/microsoft/vscode/issues/1751 still hasn't been implemented which would make this type of support much easier.

**Comment by lukaszsamson** [comment]
Fixed in https://github.com/elixir-lsp/vscode-elixir-ls/pull/258
